cloud-controller-manager/do/loadbalancers.go

```diff
@@ -1261,6 +1268,16 @@ func getEnableBackendKeepalive(service *v1.Service) (bool, error) {
 	return enableBackendKeepalive, nil
 }
 
+func getProjectID(service *v1.Service) string {
+	projectID, ok := service.Annotations[annDOLBProjectID]
+
+	if !ok {
+		return ""
```

Comment:
Not sure currently if there is a way to obtain the cluster project id and use that as the default. 

As it stands this loadbalancer will be assigned to the user's default project